src/content/plugins/module-federation-plugin.mdx

```diff
@@ -34,7 +34,7 @@ module.exports = {
 
 Create a new runtime chunk with the specified name.
 
-**webpack.js.org**
+**webpack.config.json**
```

Comment:
```suggestion
**webpack.config.js**
```

Reply:
Trying to do 100 things at the same time ;)